lib/utilities/monitoringHandler.js

```diff
@@ -48,7 +56,7 @@ function monitoringHandler(clientIP, req, res, log) {
     function monitoringEndHandler(err, results) {
         writeResponse(res, err, results, error => {
             if (error) {
-                return log.end().warn('monitoring error', { err: error });
+                return log.end().warn('monitoring error', { err: error, error: results });
             }
```

Comment:
Right now, we're logging both err and error as the same object. It would be clearer to separate them if they represent different errors.

Reply:
This is to prevent changing the previous behavior.

- `err` is the error, so for arsenal error it will be like: `{"MethodNotAlloed": true}`, but for regular `Error` like the one catched here it will be `{}`
- `error` is the body passed to the response, so for arsenal error, any error managed by us it's `[]`, but specifically for the caugh prom error, I pass the message `error.toString()` as result in body so it will be `{"message":"Error: Operation timed out."}` because nodejs `Error` and `ArsenalError` logging don't mix well.

I added this so we can see if the metric error is something like `Operation timed out` or `write EPIPE` or any other cluster internal problem

Reply:
okay, I understand that.
Can we make the code more readable, for example right now error is a key and a variable in the same scope?

I also prefer now having an `err:` and in the same line `error` unless I am missing something, there is no specific definition for err and error, so worth renaming them so CS or an admin can debug and understand.

Reply:
I could make something like `{message: err.toString(), ...err}`

For arsenal error it will look like `"err":{"MethodNotAllowed": true, "message": "This method is not allowed on this resource"}`

And for `Error` it will look like `"err":{"message":"Error: Operation timed out."}`

Reply:
Thanks, that sounds good to me.
Considering this approving the PR.